### PR TITLE
[GLUTEN-7641][VL] Add perf analysis scripts for TPCH workload

### PR DESCRIPTION
**Pull request by marin-ma**

### CODE REVIEW

File: tools/workload/benchmark_velox/README.md [hunk 1]
```diff
@@ -36,3 +36,23 @@ We also provide a script [run_tpc_workload.sh](./run_tpc_workload.sh). This scri
...
+
+The notebook also produces a trace-viewer JSON file to analyze workload statistics. This includes SAR metrics and stage/task-level breakdowns. Using this tool, users can compare statistics across stages and queries, identify performance bottlenecks, and target specific stages for optimization.
+
+If you have set up and launched Catapult trace-viewer server (refer to the **Set up perf analysis tools (optional)** section in [initialize.ipynb](./initialize.ipynb)), you can explore a sample trace-viewer JSON file. To do so: 
+
+1. Copy the sample file [trace_result_tpch_q1.json](./sample/trace_result_tpch_q1.json) to the `$HOME/trace_result` directory
+2. Open the following link in your browser to view the results: http://[your-host-ip]:1088/tracing_examples/trace_viewer.html#/tracing/test_data/trace_result_tpch_q1.json
+
+This visualization helps to better understand performance metrics and optimize accordingly.
```
Comment (FelixYBW) [inline tools/workload/benchmark_velox/README.md:58]: Can you add one more guide that the json file can be open by chrome about://tracing?

**Comment by FelixYBW** [comment]
Let's use the perf stat tool to add PMU events as well.

File: tools/workload/benchmark_velox/analysis/requirements.txt
Comment (weiting-chen) [inline tools/workload/benchmark_velox/analysis/requirements.txt:1]: Any description missing in the document?
I cannot find in which steps will use this file to install the requirements.

Comment (weiting-chen) [reply tools/workload/benchmark_velox/analysis/requirements.txt:1]: Got it in tools/workload/benchmark_velox/initialize.ipynb, please ignore this comments. Thanks.

File: tools/workload/benchmark_velox/native_sql_initialize.ipynb [hunk 2]
```diff
@@ -788,27 +818,23 @@
...
-    "            else:\n",
-    "                output_nb_name = output_nb\n",
-    "            output_nb_dir = os.path.dirname(output_nb_name)\n",
-    "            server_nb_dir = os.path.join('PAUS', self.base_dir, output_nb_dir)\n",
-    "            !ssh {self.server} \"mkdir -p {server_nb_dir}\"\n",
-    "            !scp {output_nb} {self.server}:{server_nb_dir}\n",
-    "            self.finished_nb = f\"http://{self.server}:8888/tree/{self.base_dir}/{output_nb_name}\"\n",
+    "            if output_nb.startswith(cwd):\n",
+    "                output_nb = os.path.relpath(output_nb, home)\n",
+    "                self.finished_nb = f\"http://{localhost}:8888/tree/{output_nb}\"\n",
```
Comment (weiting-chen) [inline tools/workload/benchmark_velox/native_sql_initialize.ipynb:834]: Just make sure this is not a typo.
I can find there are two jupyter notebook will be launched(8888 and 8889) in the initialize.ipynb.
8888 for user usage, 8889 for perf. analysis result, right?

Comment (marin-ma) [reply tools/workload/benchmark_velox/native_sql_initialize.ipynb:834]: Yes, port 8888 is used for accessing benchmark scripts and 8889 for perf analysis scripts. We need to start the jupyter server for perf analysis script under virtual environment, as described in initialized.ipynb, therefore they cannot share the same port.

**Comment by FelixYBW** [comment]
Let me try soon

**Pull request merged**